src/styles/index.ts

```diff
@@ -2762,6 +2762,17 @@ const styles = (theme: ThemeColors) =>
             ...cursor.cursorDefault,
         },
 
+        topUnreadIndicatorContainer: {
+            position: 'relative',
+            width: '100%',
+            height: 17,
```

Comment:
Where does this number come from exactly? Just asking since we typically do things with base4 (so 16 or 20)

Reply:
>Where does this number come from exactly? Just asking since we typically do things with base4 (so 16 or 20)

@shawnborton I _think_ this height is coming from the 1px rule with 8px of padding on top and bottom. (So 1+8+8=17)

Reply:
Got it, that makes sense - thanks for confirming!

Reply:
Maybe it would be useful to add this comment right about the `topUnreadIndicatorContainer` style rule so other people also don't get confused by this num?